frontend/lib/src/components/elements/ArrowTable/styled-components.ts

```diff
@@ -41,12 +32,53 @@ export const StyledTable = styled.table(({ theme }) => ({
     color: theme.colors.fadedText60,
     textAlign: "left",
     wordWrap: "break-word",
+    display: "inline-block",
+  },
+}))
+
+export const StyledTableBorder = styled.div(({ theme }) => ({
+  // Add the enclosing border on an extra wrapper around the table. This ensures that
+  // when the table scrolls horizontally on small windows, it still shows a border all
+  // around the table and the table doesn't look cut off.
+  border: `${theme.sizes.borderWidth} solid ${theme.colors.borderColorLight}`,
+  borderRadius: theme.radii.default,
+  overflow: ["auto", "overlay"],
+}))
+
+export const StyledTable = styled.table(({ theme }) => ({
+  width: theme.sizes.full,
+  color: theme.colors.bodyText,
+
+  borderSpacing: 0,
+
+  // Add border radius to corner cells to prevent colored background from showing
+  // through the border.
+  "& thead tr:first-child th:first-child": {
+    borderTopLeftRadius: theme.radii.default,
+  },
+  "& thead tr:first-child th:last-child": {
+    borderTopRightRadius: theme.radii.default,
+  },
+  "& tbody tr:last-child td:first-child": {
+    borderBottomLeftRadius: theme.radii.default,
+  },
+  "& tbody tr:last-child td:last-child": {
+    borderBottomRightRadius: theme.radii.default,
```

Comment:
This might not be needed anymore with the change to use the wrapping border

Reply:
Your're right, removing.